nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>s/show_docs/show_doc</p><p> </p><p>also "instead of showing docs" it is "in addition to show_doc, the source code is also displayed, for example..."</p><p> </p><p>This is because show_doc is still run and is showing the source code as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>